network/struct.go

```diff
@@ -17,80 +17,80 @@ import (
 	"golang.org/x/net/context"
 )
 
-// MaxRetryConnect defines how many times should we try to connect
+// MaxRetryConnect defines how many times we should try to connect.
 const MaxRetryConnect = 10
 
-// MaxIdentityExchange is the maximum time waited for an exchange of
-// the identity to happen.
+// MaxIdentityExchange is the timeout for an identityExchange.
 const MaxIdentityExchange = 5 * time.Second
 
-// WaitRetry defines how much time should we wait before trying again
+// WaitRetry is the timeout on connection-setups.
 const WaitRetry = 100 * time.Millisecond
 
 // The various errors you can have
 // XXX not working as expected, often falls on errunknown
 
-// ErrClosed is when a connection has been closed
+// ErrClosed is when a connection has been closed.
 var ErrClosed = errors.New("Connection Closed")
 
-// ErrEOF is when the EOF signal comes to the connection (mostly means that it
-// is shutdown)
+// ErrEOF is when the connection sends an EOF signal (mostly because it has
+// been shut down).
 var ErrEOF = errors.New("EOF")
 
-// ErrCanceled means something went wrong with the sending or receiving
+// ErrCanceled means something went wrong in the sending or receiving part.
 var ErrCanceled = errors.New("Operation Canceled")
 
-// ErrTemp is a temporary error
+// ErrTemp is a temporary error, recovery possible.
 var ErrTemp = errors.New("Temporary Error")
```

Comment:
Thanks for the clarification! I'm still not sure if it's a good idea that we mimic them after all... But yeah, if this currently isn't used (or only in the tests), you can remove it. Because we do not seem to handle it/work with it.